data.js

```diff
@@ -0,0 +1,33 @@
+export const data = {
+  "header": {
+    "title-team": "The creative crew",
+    "description-team-title": "Who we are",
+    "description-team-text": "We are team of creatively diverse. driven. innovative individuals working in various locations from the world."
+  },
+
```

Comment:
You can delete these blank lines if you want

Reply:
fac9e7d